src/lightning_lite/__setup__.py

```diff
@@ -60,7 +61,27 @@ def _adjust_manifest(**__: Any) -> None:
         fp.writelines(lines)
 
 
+def _check_install_collision() -> None:
+    try:
+        import pytorch_lightning
```

Comment:
Turns out we cannot import like this unless we use `pip install --no-build-isolation`: https://pip.pypa.io/en/stable/cli/pip_install/#cmdoption-no-build-isolation

I'm going to try with a subprocess call

Reply:
```py
        version = subprocess.check_output(
            "python -c 'import pytorch_lightning; print(pytorch_lightning.__version__)'",
            shell=True,
        )
```

Suffers from the same issue.

I don't think we can reliably check this.